src/NuGet.CommandLine/Commands/RestoreCommand.cs

```diff
@@ -148,8 +148,10 @@ public RestoreCommand()
         private async Task<bool> PerformNuGetV3RestoreAsync(string packagesDir, string inputPath)
         {
             var inputFileName = Path.GetFileName(inputPath);
-            PackageSpec packageSpec;
+            var projectDirectory = Path.GetDirectoryName(Path.GetFullPath(inputPath));
```

Comment:
This was existing code in this file that moved. I'm not sure that it isn't needed.